root/client/src/components/user/dashboard.js

```diff
@@ -1,18 +1,52 @@
-import { React, useState } from 'react';
+import { React, useState, useEffect } from 'react';
 import { PollForm } from '../poll/pollForm';
 
 export const Dashboard = () => {
   const [showPollForm, setShowPollForm] = useState(false);
+  const [polls, setPolls] = useState([]);
   const handleSubmit = async (e, formData) => {
     e.preventDefault();  
     await fetch('/polls', { method: 'POST', headers: { 'Content-Type': 'application/json' }, body: JSON.stringify(formData) })
   }
 
+  useEffect(() => {
+    const getPolls = async () => {
+      return await fetch('/polls/all').then(data => data.json()).then(result => result.response);
+    }
```

Comment:
does the immediate function return not work here?

```suggestion
    const getPolls = async () => await fetch('/polls/all').then(data => data.json()).then(result => result.response);
```